### ISSUE TITLE
How to ipv4 and ipv6 on self-hosted?

### ISSUE THREAD

**Issue by barart** [PeterDaveHello/ipinfo.tw#35]
I just pull the docker image, but, how do i set by default ipv4 over ipv6? or what is the endpoint for each one?

**Comment by PeterDaveHello** [comment]
Can you provide more details about your deployment environment?

**Comment by WilliDieEnte** [comment]
The Question is simply how to _only_ request IPv4 or IPv6.
IPify for example has [api6.ipify.org](https://api6.ipify.org) for IPv6 only and [api4.ipify.org](https://api4.ipify.org) for IPv4 only.
The person only asks how this would be possible with ipinfoTW ^^
Some clients have both v4 and v6, but it would be better to get the v4 or v6 in some situations.
(one of those idots needing that would be me xd)

**Comment by PeterDaveHello** [comment]
Make use of the corresponding domain that has only A or AAAA records for IPv4 and IPv6, which should be good enough for it.

**Comment by barart** [comment]
No, i mean, if you go directly to ipinfo.tw, by default it shows you your ipv6; how do i make that by default it shows my ipv4 instead ipv6?
it would be nice if instead of subdomains (ipv4.xxx and ipv6.xxx) you can use a endpoint like /v4 or /v6 and if none version is supplied it show both (ipv4 and ipv6)
